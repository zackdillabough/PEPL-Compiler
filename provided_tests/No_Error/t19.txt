POLY A(X) = X + 1;
POLY B(Y) = Y + 2;
POLY C(Z) = Z - 3;
POLY D(f, g) = f^2 - g + 1;
START
INPUT P;
INPUT Q;

A(B(C(A(B(C(A(B(C(A(B(C(A(B(C(A(B(C(A(B(C(A(B(C(A(B(C(A(B(C(P))))))))))))))))))))))))))))));

INPUT P;
INPUT Q;
D(P, D(0, 1));
D(P, D(Q, D(P, D(Q, D(P, D(Q, D(P, Q)))))));

INPUT P;
INPUT Q;

A(B(C(A(B(C(A(B(C(A(B(C(A(B(C(A(B(C(A(B(C(A(B(C(A(B(C(A(B(C(P))))))))))))))))))))))))))))));

INPUT P;
INPUT Q;

D(P, D(Q, D(P, D(Q, D(P, D(Q, D(P, Q)))))));

INPUT P;
INPUT Q;

A(B(C(A(B(C(A(B(C(A(B(C(A(B(C(A(B(C(A(B(C(A(B(C(A(B(C(A(B(C(P))))))))))))))))))))))))))))));

INPUT P;
INPUT Q;

D(P, D(Q, D(P, D(Q, D(P, D(Q, D(P, Q)))))));

INPUT P;
INPUT Q;

A(B(C(A(B(C(A(B(C(A(B(C(A(B(C(A(B(C(A(B(C(A(B(C(A(B(C(A(B(C(P))))))))))))))))))))))))))))));

INPUT P;
INPUT Q;

D(P, D(Q, D(P, D(Q, D(P, D(Q, D(P, Q)))))));

34 12 36 71 27 21 9 6 0 34 12 36 71 27 21 9 6 0 34 12 36 71 27 21 9 6 0
