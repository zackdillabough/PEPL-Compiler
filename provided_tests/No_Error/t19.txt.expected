34
1297
-10009
27
211
0
-3347
71
1513
